lib/Common/Memory/AllocatorTelemetryStats.h

```diff
@@ -0,0 +1,25 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include "Common/Tick.h"
+
+struct AllocatorDecommitStats
+{
+    Js::Tick lastLeaveDecommitRegion;
+    Js::TickDelta maxDeltaBetweenDecommitRegionLeaveAndDeCommit;
```

Comment:
DeCommit => Decommit based on casing in the surrounding lines?

Reply:
fixed